dotnet/src/SemanticKernel/CoreSkills/ConditionalSkill.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Xml;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.KernelExtensions;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Planning;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.CoreSkills;
+public class ConditionalSkill
```

Comment:
Need an empty line between namespace and class declaration. Also would be nice to add XML comments. Same applies for [ConditionException.cs](https://github.com/microsoft/semantic-kernel/pull/175/files#diff-09fc23f0901f6a82ceeeaece2da00de5d9784ca91de8116aafce3a2893a2a5e8)